### PR TITLE
Zoom.Zoom version 5.11.8200

### PR DESCRIPTION
**Pull request by mivsvit**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/76559)

### CODE REVIEW

File: manifests/z/Zoom/Zoom/5.11.10.8200/Zoom.Zoom.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.11.10.8200
```
Comment (ItzLevvie) [inline manifests/z/Zoom/Zoom/5.11.10.8200/Zoom.Zoom.installer.yaml:5]:
```suggestion
PackageVersion: 5.11.8200
```
This must match Control Panel.

File: manifests/z/Zoom/Zoom/5.11.10.8200/Zoom.Zoom.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.11.10.8200
```
Comment (ItzLevvie) [inline manifests/z/Zoom/Zoom/5.11.10.8200/Zoom.Zoom.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.11.8200
```
This must match Control Panel.

File: manifests/z/Zoom/Zoom/5.11.10.8200/Zoom.Zoom.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.11.10.8200
```
Comment (ItzLevvie) [inline manifests/z/Zoom/Zoom/5.11.10.8200/Zoom.Zoom.yaml:5]:
```suggestion
PackageVersion: 5.11.8200
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by mivsvit** [comment]
Closing request and resubmitting with updated version number.

**Pull request closed without merging**